components/dot_menu/dot_menu.jsx

```diff
@@ -301,6 +305,19 @@ export default class DotMenu extends React.PureComponent {
                 );
             });
 
+        const pluginIntegrationsItems = this.props.pluginIntegrations.filter((item) => {
+            return item.location === 'POST_ACTION';
```

Comment:
Can we have these `item.location` values in a typescript enum? Possibly in mm-redux if it makes sense.

```ts
enum PluginIntegrationLocation {
    POST_ACTION = 'POST_ACTION',
}
```

Actually I just noticed this file is not typescript'd. Maybe have it as a constant like action types are usually done?